Vagrantfile

```diff
@@ -351,13 +351,16 @@ Vagrant.configure('2') do |config|
     cfg.vm.guest = :windows
     # config below prevents the installation of latest Chef on the box.
     # Reference: https://github.com/chef/vagrant-omnibus/issues/118
-    config.omnibus.install_url = 'https://packages.chef.io/files/stable/chef/14.5.27/windows/2012r2/chef-client-14.5.27-1-x64.msi'
+    # TODO: figure out how to install chef 15 on a windows box
+    cfg.omnibus.install_url = 'https://packages.chef.io/files/stable/chef/14.5.27/windows/2012r2/chef-client-14.5.27-1-x64.msi'
     cfg.vm.provider :virtualbox do |vb|
       vb.customize ['modifyvm', :id, '--memory', 1024]
     end
     cfg.vm.provision :chef_client do |chef|
       chef_defaults chef, :f_win2012r2, 'splunk_standalone'
+      chef.arguments = ''
```

Comment:
This was missed in #213 and I haven't been able to get it working with Chef 15 yet. So for now I left it on 14 so I could test the other changes I actually wanted to make.